index.html

```diff
@@ -27,125 +33,273 @@ <h3>I’am a Software Developer</h3>
+            <li class="transla"><img src="pictures/translu2.png" alt="" /></li>
+          </ul>
         </div>
-      </section>
+        <div class="activ-g first-box1">
+          <h6 class="en">Profesional Art<br />Printing Data <br /></h6>
+          <p class="text2">
+            A daily selection of privately <br />
+            personalized reads; no accounts or <br />
+            sign-ups required. has been the <br />
+            industry's standard
+          </p>
+          <div class="">
+            <ul class="linksd">
+              <li class="transla"><img src="pictures/translu.png" alt="" /></li>
+              <li class="transla">
+                <img src="pictures/translu1.png" alt="" />
+              </li>
+              <li class="transla">
+                <img src="pictures/translu2.png" alt="" />
+              </li>
+            </ul>
+          </div>
+          <div class="oragan">
+            <button class="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="d-flex">
+        <div class="box-type-3 first-box1 name-desk">
+          <h2>
+            Profesional Art <br />
+            Printing Data More
+          </h2>
+          <p>
+            A daily selection of privately personalized <br />
+            reads; no accounts or sign-ups required. <br />
+            has been the industry's standard
+          </p>
+          <ul class="linksd">
+            <li class="transla"><img src="pictures/translu.png" alt="" /></li>
+            <li class="transla"><img src="pictures/translu1.png" alt="" /></li>
+            <li class="transla"><img src="pictures/translu2.png" alt="" /></li>
+          </ul>
+        </div>
+        <div class="first-box1 name-desk">
+          <h2>Data Dashboard <br />Healthcare</h2>
+          <p>
+            A daily selection of privately personalized <br />
+            reads; no accounts or sign-ups required. <br />
+            has been the industry's standard
+          </p>
+          <ul class="linksd">
+            <li class="transla"><img src="pictures/translu.png" alt="" /></li>
+            <li class="transla"><img src="pictures/translu1.png" alt="" /></li>
+            <li class="transla"><img src="pictures/translu2.png" alt="" /></li>
+          </ul>
+        </div>
+        <div class="first-box1 name-desk">
+          <h2>Data Dashboard <br />Healthcare</h2>
+          <p>
+            A daily selection of privately personalized <br />
+            reads; no accounts or sign-ups required. <br />
+            has been the industry's standard
+          </p>
+          <ul class="linksd">
+            <li class="transla"><img src="pictures/translu.png" alt="" /></li>
+            <li class="transla"><img src="pictures/translu1.png" alt="" /></li>
+            <li class="transla"><img src="pictures/translu2.png" alt="" /></li>
+          </ul>
+        </div>
+      </div>
```

Comment:
### Optional
- [ ] I can't see project images. They are in there but not showing. Something wrong in your folder path. I am sure it is working  your side. But it is has to work every pc.  You can check with this [website](https://raw.githack.com)

![image](https://user-images.githubusercontent.com/38635158/126019195-2891fdf3-1ae6-4e73-b0c2-07751ccf5460.png)